samples/net/sockets/http_server/Kconfig

```diff
@@ -68,4 +68,9 @@ config NET_SAMPLE_WEBSOCKET_STATS_INTERVAL
 	  This interval controls how often the net stats data shown on the web page
 	  will be updated.
 
+# Enable the build of PSA Crypto core provided by Mbed TLS for platforms that
+# do not have TF-M.
+config MBEDTLS_PSA_CRYPTO_C
```

Comment:
If this is only used for this sample, then please namespace the option name by `NET_SAMPLE` to make it clear that this is not a global option.

Reply:
Actually this is the very same Kconfig defined in `zephyr/modules/mbedtls/Kconfig.tls-generic`. I added it here as trick to enable it for platforms that do not have TF-M support.

As stated [previously](https://github.com/zephyrproject-rtos/zephyr/pull/85201#issuecomment-2636395793) we might improve how PSA Crypto support is enabled, but that will require additional analysis and work and it's out of scope for this PR. 

The [other option](https://github.com/zephyrproject-rtos/zephyr/pull/85201#issuecomment-2636185631) was to set `CONFIG_MBEDTLS_PSA_CRYPTO_C=y` directly in the `prj.conf` for all the samples, but that would result in warnings at compile time for boards that support TF-M, so I preferred not to go that way.

Reply:
This looks very hackish now, basically you are stealing Kconfig option from mbedtls which I would say is a big no-no.